svelte-kit-scss/src/lib/components/Profile/OrganizationProfile/OrganizationProfile.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { beforeEach, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import OrganizationProfile from './OrganizationProfile.svelte';
+import { MOCK_GITHUB_ORGANIZATION_PROFILE_INFORMATION } from '$lib/helpers/mocks/users';
+import { remapPublicProfileInformation } from '$lib/helpers';
+
+const mockOrgProfile = remapPublicProfileInformation(MOCK_GITHUB_ORGANIZATION_PROFILE_INFORMATION);
+
+describe('OrganizationProfile', () => {
+  beforeEach(() => {
```

Comment:
Seeing we're not modifying or interacting with the component, we can use `beforeAll` here to speed up the tests.